memory/memhotplug.py

```diff
@@ -189,14 +189,14 @@ def test_hotplug_toggle(self):
 
     def test_dlpar_mem_hotplug(self):
         if 'ppc' in platform.processor() and 'PowerNV' not in open('/proc/cpuinfo', 'r').read():
-            if "mem_dlpar=yes" in process.system_output("drmgr -C", ignore_status=True, shell=True):
+            if b"mem_dlpar=yes" in process.system_output("drmgr -C", ignore_status=True, shell=True):
```

Comment:
wouldn't it be better to compare as string ?

Reply:
I am not sure, because we return bytes with `process.system_output`. Either it has to be decoded as string or used as is as bytes, both works though. Is it better to decode and use as string?

Reply:
To minimise unknown breakages and also maintain a uniformity, it is better we decide either string to byte conversion or the inverse. And since all our tests are working with string better, I thought that would be better. There is nothing else suggesting which is better.

And this was just a suggestion. I am fine with either approach.

Reply:
i vote for `string to byte conversion ` as in avocado lib also we follow the same